composer.json

```
@@ -8,7 +8,7 @@
         "ext-json": "*",
         "api-platform/api-pack": "^1.2",
         "easycorp/easyadmin-bundle": "^2.1",
-        "paypal/paypal-checkout-sdk": "1.0.0",
+        "paypal/paypal-checkout-sdk": "^1.0",
```

```suggestion
        "paypal/paypal-checkout-sdk": "^1.0",
        "paypal/paypalhttp": "^1.0",
```
Why you don't require paypalhttp if it required by paypal-checkout-sdk ?